css/style.css

```diff
@@ -0,0 +1,2063 @@
+@font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 400;
+    src: local('Montserrat Regular'), local('Montserrat-Regular'),
+        url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2'), 
+        url('../fonts/montserrat-v14-latin_cyrillic-regular.woff') format('woff'); 
+    }
+
+    @font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 500;
+    src: local('Montserrat Medium'), local('Montserrat-Medium'),
+        url('../fonts/montserrat-v14-latin_cyrillic-500.woff2') format('woff2'),
+        url('../fonts/montserrat-v14-latin_cyrillic-500.woff') format('woff'); 
+    }
+
+    @font-face {
+    font-family: 'Montserrat';
+    font-style: normal;
+    font-weight: 600;
+    src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'),
```

Comment:
Codacy found an issue: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)